stories/Atom/Buttons-and-labels/Cta_button/buttons.scss

```diff
@@ -0,0 +1,145 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/breakpoints";
+
+.btn {
+    border: none;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 1;
+    letter-spacing: 0.03em;
+    color: $color-white;
+    cursor: pointer;
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    font-family: $Font-Family-ProximaRegular;
+    padding: $spacing-16 $spacing-24;
+    @include devicebreak(small) {
+        min-width: 168px;
+        display: inline-flex
+    }
+}
+
+.btn-primary {
+    background-color: $color-dark-red;
+    &:hover {
+    background-color: $color-red;
+    }
+}
+
+.btn-secondary {
+    padding: 18px $spacing-24;
+    background-color: $color-blue-600;
+    &:hover {
+        background-color: $color-blue-400;  
+    }
+}
+
+.btn.btn-arrow.disabled {
+    background-color: $color-gray-400;   
+    color: $color-gray-300;
+    pointer-events: none;
+    &::after {
+        content: "";
+        width: 13px;
+        height: 20px;
+        background: url(#{$img-path-Icon}/chevron-right-white.svg)
+        no-repeat left center;
+    }
+}
+
+.btn.btn-no_arrow.disabled, .btn.btn-secondary.disabled {
+    background-color: $color-gray-400;   
+    color: $color-gray-300;
+    pointer-events: none;
+}
```

Comment:
@pallavikadam-srijan We can keep the the styling for disabled same, please don't repeat the styling for the other classes